README.md

```diff
@@ -39,7 +39,7 @@ Get a live preview of the project on this link
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/hunter4466/Math_magicians/issues).
+Feel free to check the issues page. [issues page](https://github.com/hunter4466/Math_magicians/issues).
```

Comment:
- [ ] [OPTIONAL] You have added a link which is broken. Please have a look at it.